docs/admin/eventing/broker-configuration.md

```diff
@@ -13,7 +13,7 @@ metadata:
   labels:
     eventing.knative.dev/release: devel
 data:
-  # Configuration for defaulting Channels that do not specify CRD implementations.
+  # Configuration for defaulting Brokers that do not specify a spec.config or Broker class.
```

Comment:
@abrennan89 in response to your request for changes in the aforementioned PR:

---

```diff
- for defaulting Brokers
+ for default Brokers
```

The configuration is used for defaulting _all_ Brokers. It is not for the _default Broker_.

---

> I think use `default-br-config` throughout instead of `spec.config`

I think there is a misunderstand about what the comment says here. "Brokers that do not specify [...]": we are talking about what *Brokers* specify (or don't specify) via their *spec.config* field, not what the ConfigMap specifies.

(Here is how Eventing works: if ` spec.config` is missing, Brokers get their config from the ConfigMap. Otherwise not.)

---

tl;dr; The wording is correct here.

Reply:
Not sure if it might make sense to say something like:
"Configures the default for any Broker that does not specify a `spec.config` or Broker class."

It might make it clearer that it's not referring to the _default Broker_ but to _any_ Broker.

Reply:
That sounds clearer yes. I submitted this suggestion to Eventing and will update this PR once the change is approved: https://github.com/knative/eventing/pull/5713